### PR TITLE
Add doc about advanced char sequence generation

### PR DESCRIPTION
**Pull request by snuyanzin**

### CODE REVIEW

File: docs/documentation/password.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Now it allows to specify different set of symbols which should be contained and minimum amount of symbols per such set.
+
+Imagine there is a need to generate a password with length 8 and containing minimum 3 digits and 2 upper case symbols from en locale.
+=== "Java"
+
+    ``` java
+         var faker = new Faker();
+         String password = faker.text().text(Text.TextSymbolsBuilder.builder()
+                             .len(8)
+                             .with(EN_UPPERCASE, 3)
```
Comment (kingthorin) [inline docs/documentation/password.md:13]: 2 like the lead-in says?

**Comment by kingthorin** [comment]
Downstream tests are failing but the issue seems unrelated to this PR

File: docs/documentation/password.md (hunk 1 above)
Comment (snuyanzin) [reply docs/documentation/password.md]: yep, thanks for noticing, fixed

**Comment by snuyanzin** [comment]
>Downstream tests are failing but the issue seems unrelated to this PR

yeahh... that was the reason https://github.com/datafaker-net/datafaker-gen/pull/12
we added connector to BigQuery and in tests it requires service account...

Anyway we don't need such test on datafaker side, so just skip it within https://github.com/datafaker-net/datafaker/pull/1247

**Review by kingthorin — APPROVED** [review]

**Pull request merged**